microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/jaxrs/java/JaxRsCodeLensParticipant.java

```diff
@@ -133,7 +133,7 @@ private static void collectURLCodeLenses(IJavaElement[] elements, String rootPat
 				// @DELETE, @PUT
 				// JAX-RS
 				// annotation
-				if (isJaxRsRequestMethod(method) && Flags.isPublic(method.getFlags())) {
+				if (isClickableJaxRsRequestMethod(method) && Flags.isPublic(method.getFlags())) {

```

Comment:
What do you think about also providing the unclickable urls for the application's own REST endpoints in this PR? When I read the issue initially, that's what I thought it was talking about.

Reply:
I think it makes sense to do that as well. @rgrunber what do you think?